.gitignore

```diff
@@ -2,6 +2,7 @@ dist
 yarn-error.log
 node_modules
 coverage
+.DS_store
```

Comment:
I battled a lot (https://twitter.com/search?l=&q=%22~%2F.gitignore%22%20from%3Avvoyer&src=typd) around global gitignore too (for instance: node_modules) but in the end given that the developer community is very different, it's just easier to put it and forget about a one line addition (versus the added convo).

I know, we can't have nice things :D